access/src/main/java/edu/unc/lib/dl/ui/controller/FullRecordController.java

```diff
@@ -162,8 +163,13 @@ public String getFullRecord(String pidString, Model model, HttpServletRequest re
 
         AccessGroupSet principals = getAgentPrincipals().getPrincipals();
 
-        aclService.assertHasAccess("Insufficient permissions to access full record",
-                pid, principals, Permission.viewMetadata);
+        String errorMsg = "Insufficient permissions to access full record";
```

Comment:
Put this directly into the assertHasAccess call and remove it from the log statement. At the moment, my log is getting it 3 times:
```
e.u.l.d.u.c.FullRecordController - Insufficient permissions to access full record. Insufficient permissions to access full
 record
edu.unc.lib.dl.acl.exception.AccessRestrictionException: Insufficient permissions to access full record
```